front-end/src/api/index.js

```diff
@@ -0,0 +1,53 @@
+import { ApolloClient, InMemoryCache } from "@apollo/client";
```

Comment:
Ek het die `graphQL` folder rename na `api`. Die rede is dat die data wat jy van die server af kry is basies jou "API". Om dit `graphQL` te noem expose die implementation detail wat nie great is as jy dit sou wou verander in die toekoms nie.

Wat jy baie goed gedoen het hier is die abstraction layer en jou naming soos `fetchCurrenciesAPI`. Dit beteken dat sê nou jy moet in die toekoms eerder REST API gerbuik (of enige ander tech as graphQL) kan jy dit uit swop in jou `api` folder en die res van die app hoef nooit eers daarvan te weet nie